.travis.yml

```diff
@@ -146,11 +146,10 @@ before_script:
     - set -o errexit; source .travis/before_script.sh
 script:
     - export CONTINUE=1
+<<<<<<< HEAD
```

Comment:
Merge error!

Reply:
Ouch, Will fix way to many rebases and interleaved PRs

Reply:
fixed